### PR TITLE
Filter untested datapoints docs

### PR DESCRIPTION
**Pull request by mkaramlou**
### Linked issue(s)
KOL-7249

### Please check if the PR fulfills these requirements

- [ ] Include reference to internal ticket and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Relevant tests for the changes have been added
- [ ] Relevant docs have been added / updated

### CODE REVIEW

File: docs/dataset/core-concepts/index.md [hunk 1]
```diff
@@ -130,6 +130,10 @@ which provides a quick and standardized high level overview of which models perf
 For step-by-step instructions, take a look at the quickstart for
 [model comparison](../quickstart.md/#step-5-compare-models).
 
+!!! tip
+    Use the `Filter Untested Datapoints` option to narrow down your metrics to only include datapoints that all selected
```
Comment (munkyshi) [inline docs/dataset/core-concepts/index.md]: Should we call this `Filter to Intersection` (and below)? `Filter Untested Datapoints` only applies in contexts where there is a single model.

File: docs/dataset/core-concepts/index.md [hunk 2]
```diff
@@ -130,6 +130,10 @@ which provides a quick and standardized high level overview of which models perf
...
+    models have tested on. This allows an apple to apple comparison of metrics.
```
Comment (munkyshi) [inline docs/dataset/core-concepts/index.md]:
```suggestion
    models have tested on. This allows for an apple to apple comparison of metrics.
```

File: docs/dataset/quickstart.md [hunk 3]
```diff
@@ -207,6 +207,10 @@ Metrics in this view will also be highlighted according to how much they improve
 reference model. In this example, RetinaFace performs better than the random model over the entire dataset
 as well as over all test cases we defined earlier.
 
+!!! tip
+    Use the `Filter Untested Datapoints` option to narrow down your metrics to only include datapoints that all selected
+    models have tested on. This allows an apple to apple comparison of metrics.
```
Comment (munkyshi) [inline docs/dataset/quickstart.md]:
```suggestion
    models have tested on. This allows for an apple to apple comparison of metrics.
```

File: docs/dataset/core-concepts/index.md [hunk 4]
```diff
@@ -158,6 +162,10 @@ uploaded model results in order of their rank.
 from different distributions. Kolena uses the range of each metric to
 estimate its standard deviation which is used in calculating the z-score.
 
+**Filter Untested Datapoint**: this option allows you to rank your models only on
+datapoints that they were all tested on. Use this feature if its important to you to rank
+your models based on the datapoints they were tested on.
```
Comment (munkyshi) [inline docs/dataset/core-concepts/index.md]: Thoughts?
```suggestion
**Filter Untested Datapoint**: this option allows you to rank your models only on
datapoints that they were all tested on. Use this feature if its important to you to
standardize the comparison set of datapoints.
```

File: docs/dataset/core-concepts/index.md (hunk 1 above)
Comment (mkaramlou) [reply docs/dataset/core-concepts/index.md]: Yeah - I think we started with "filter to intersection" and changed to this. We can rename to "use common datapoints"?

**Review by munkyshi — APPROVED** [review]

**Pull request merged**